monero-outreach-docs/translations/archive/wallet_primer_fr.md

```diff
@@ -0,0 +1,90 @@
+
+### _Sécurité lors de la génération de portefeuille_
+Des précautions particulières doivent être prises lors de la création de vos clés avec les méthodes décrites ci-dessus. Une monnaie cryptographique privée peut être volée, car c’est facile à dépenser, est de nature immatériel, intraçable, et ne requiert aucune présence physique pour être véhiculée. Soyez prudent, sinon quelqu’un prendra vos Monero. Toutefois la raison doit être utilisée. Tenez compte du fait que si vous avez peu d’argent liquide à dépenser, vous le porterez dans un portefeuille classique, alors que les gros bénéfices d’une entreprise sont transportés dans des véhicules blindés. Vous devez considérer le niveau de sécurité pour votre portefeuille papier de Monero de la même manière, et mesurer l’effort mis dans la sécurité en proportion de vos avoirs. 
+Pour un niveau de sécurité équivalent à celui d’un portefeuille classique, vous pouvez utiliser votre téléphone portable ou un ordinateur classique. Il y a un débat sur ce qui est le plus sure, mais pour un niveau d’avoirs du niveau de celui d’un portefeuille, c’est probablement suffisant d’utiliser l’un ou l’autre tout en prenant des précautions basiques. Si vous utilisez un téléphone portable, démarrez un logiciel anti-virus et déconnectez l’ordinateur d’internet lorsque vous gênerez vos portefeuilles. N’utilisez aucun appareil qui vous parait suspect. Avec ces étapes vous pouvez appliquer les techniques décrites dans la précédente section afin de générer votre portefeuille papier avec un niveau de sécurité équivalent à un portefeuille classique. 
+Pour un niveau de sécurité équivalent à un véhicule blindé, vous devez utiliser un ordinateur qui ne sera plus jamais connecté à internet, et vous devez effacer sa mémoire après la génération de clés (de tel sorte que quelqu’un qui arrive à accéder physiquement à votre ordinateur ne peut pas les recréer). Presque tout le monde a un vieil ordinateur. Vous pouvez utiliser cela, ou acheter un nouvel ordinateur portable bas de gamme pour quelques centaines de dollars. Si vous imprimez vos clés de portefeuille à des fins de stockage, vous devez utiliser une imprimante sans mémoire intégrée (tel qu’une HP Deskjet 1100).  Vous devez vérifier que le hachage du logiciel tournant sur l’ordinateur servant de portefeuille (importé grâce à un CD ou une clé de stockage, car –encore une fois—il n’y a pas de connexion internet direct). Vous pouvez voir quelques exemples de tels hachages, pour le portefeuille officiel de Monero, à [getmonero.org/downloads](https://web.getmonero.org/downloads/). Quand le logiciel est signé, vous devriez vérifier la signature.
```

Comment:
1. [(de tel sorte ] ==> [ (de **telle** sorte ; accord]
2. [tel qu’une HP Deskjet] ==> [**telle** qu’une HP Deskjet ; accord]
3. [Vous devez vérifier que le hachage du logiciel tournant sur l’ordinateur servant de portefeuille] ==> [ Il manque une action, nous devons vérifier que le hachage.... ]

Reply:
For 3, there was indeed something wrong in my sentence. However, I feel that 'Il manque une action', a bit strange (perhaps because a bit long). So I changed to **Enfin, vous devez vérifier le hachage du logiciel tournant sur l’ordinateur servant de portefeuille**